### DISCUSSION TITLE
[REQUEST] Inventory Health Month To Date Report

### DISCUSSION THREAD

**Discussion by ktroxler** [amzn/selling-partner-api-models#610] [Ideas]
Hello everybody!

It seems that the Inventory Health Report within selling partner API does not have an option to pull MTD reports. As it has options to pull weekly, monthly, and quarterly. Is there any time that this will be provided via the API as we see it is part of the UI in vendor central

<img width="737" alt="Screen Shot 2022-10-21 at 9 02 21 AM" src="https://user-images.githubusercontent.com/51894654/197202101-554d5180-477a-4bf1-a2e6-aa8f01801eaf.png">

**Comment by sungolivia** [comment]
is GET_VENDOR_INVENTORY_REPORT what you are looking for?
https://developer-docs.amazon.com/sp-api/docs/report-type-values#vendor-retail-analytics-reports

**Reply by ktroxler** [reply]
That's the report that I'm looking for, but the reporting period doesn't have month to date as it does in the UI. It's just offers weekly, monthly, quarterly, and yearly. It would be great to get daily or month to date recurring available. 

<img width="806" alt="Screen Shot 2022-10-21 at 2 37 43 PM" src="https://user-images.githubusercontent.com/51894654/197266473-4d7b774b-2bd5-4835-90f8-4155beddcbfb.png">
